nsdb-core/src/main/scala/io/radicalbit/nsdb/post_proc/package.scala

```diff
@@ -49,7 +49,7 @@ package object post_proc {
       case Some(_: InternalTemporalAggregation) =>
         val temporalSortedResults = chainedResults.sortBy(_.timestamp)
         statement.limit.map(_.value).map(v => temporalSortedResults.takeRight(v)) getOrElse temporalSortedResults
-      case Some(InternalCountStandardAggregation(_, _)) if statement.order.exists(_.dimension == "value") =>
+      case Some(InternalCountStandardAggregation(_)) if statement.order.exists(_.dimension == "value") =>
```

Comment:
I don't know if we can expose an API `hasValue` here statement.order.exists(_.dimension == "value")
maybe order? statement? im pretty noob

Reply:
This condition checks if there is an `order by value` statement that needs to be managed slightly differently. Since this condition is applied only here I don't know if it's useful to provide an API, but I'm open to discussion

Reply:
👍